tests/env/node/index.js

```diff
@@ -3,5 +3,8 @@ const DefaultMeiliSearch = require('../../../dist/bundles/meilisearch.cjs.js')
 
 const CJStest = new MeiliSearch({ host:'http://localhost:7700', masterKey: 'masterKey'})
 const DefaultCJSTest = new DefaultMeiliSearch.MeiliSearch({ host:'http://localhost:7700', masterKey: 'masterKey'})
-console.log({ CJStest, DefaultCJSTest })
 
+DefaultCJSTest.generateTenantToken([]) // Resolved using the `main` field
+CJStest.generateTenantToken([]) // Resolved using the `main` field
+
+console.log({ CJStest, DefaultCJSTest })
```

Comment:
I don't follow the idea of these files `tests/env/node/index.js`, `tests/env/typescript-browser/src/index.ts` can you help me understand them?

Reply:
I created some specific environments to ensure that the bundles work correctly. 
- node: No bundler, should work with the `cjs` build
- typescript-browser, its own bundler uses the `browser` field of the SDK's package.json, so I check if it imports the right bundler and that it works the `esm` build
- typescript-node: Same as typescript-browser, but its own bundler should use the `module` field of the package.json.

During the tests I run all of their builds and if they throw an error it means there is an environment that uses the wrong bundle.